ui/app/templates/components/line-chart.hbs

```diff
@@ -26,6 +26,23 @@
   <g aria-hidden="true" class="x-axis axis" transform="translate(0, {{this.xAxisOffset}})"></g>
   <g aria-hidden="true" class="y-axis axis" transform="translate({{this.yAxisOffset}}, 0)"></g>
 </svg>
+<div data-test-annotations class="line-chart-annotations" style={{this.chartAnnotationsStyle}}>
+  {{#each this.processedAnnotations as |annotation|}}
+  <div data-test-annotation class="chart-annotation {{annotation.iconClass}} {{annotation.staggerClass}}" style={{annotation.style}}>
+      <button
+        type="button"
+        class="indicator {{if (or
+          (and this.annotationKey (eq-by this.annotationKey annotation.annotation this.activeAnnotation))
+          (and (not this.annotationKey) (eq annotation.annotation this.activeAnnotation))
+        ) "is-active"}}"
+        title={{annotation.label}}
+        onclick={{action this.annotationClick annotation.annotation}}>
+        {{x-icon annotation.icon}}
+      </button>
+      <div class="line" />
+    </div>
+  {{/each}}
```

Comment:
I’m not sure whether this would actually work but if this `#each` had a `key` would it make these elements stable? I found a focus-disappearing situation while keyboarding in a Mirage context:

![annotation-loses-focus](https://user-images.githubusercontent.com/43280/89555276-76d68e80-d7d5-11ea-8222-06a4b0b06eea.gif)

Reply:
Good call. I'll investigate.

Reply:
Fixed! Adding a `key` did the trick.